lib/Client.js

```diff
@@ -1122,6 +1122,24 @@ class Client extends EventEmitter {
         return this.requestHandler.request("PATCH", Endpoints.GUILD_TEMPLATE_GUILD(guildID, code), true, options).then((template) => new GuildTemplate(template, this));
     }
 
+    /**
+    * Update a user's voice state - See [caveats](https://discord.com/developers/docs/resources/guild#update-others-voice-state-caveats)
+    * @param {String} guildID The ID of the guild
+    * @param {Object} options The properties to edit
+    * @param {String} options.channelID The ID of the channel the user is currently in
+    * @param {Date?} [options.requestToSpeakTimestamp] Sets the user's request to speak - this can only be used when the `userID` param is "@me"
+    * @param {Boolean} [options.suppress] Toggles the user's suppress state
+    * @param {String} [userID="@me"] The user ID of the user to update
+    * @returns {Promise}
+    */
+    editGuildVoiceState(guildID, options, userID = "@me") {
```

Comment:
Consider using (guildID, memberID, options) instead.

Reply:
Options is required. Required parameter following optional parameter is also frowned upon.

Reply:
Fair enough.